### ISSUE TITLE
0.19.0 isn't on Github

### ISSUE THREAD

**Issue by adhawkins** [kalekundert/parametrize_from_file#22]
It looks like there's been a 0.19.0 release, but this hasn't made it to the releases page on GitHub.

Once this is done I'll get it packaged up for Alpine.

**Comment by adhawkins** [comment]
Any chance this could be tagged here so that I can arrange a release for Alpine?

**Comment by kalekundert** [comment]
Sorry, I let this slip past my radar.  I'll try to take care of this tonight.

**Comment by kalekundert** [comment]
Sorry, I still haven't been able to figure this out.  I'm still working on it, though.

**Comment by adhawkins** [comment]
Isn't it just a question of:

```
git tag 0.19.0 69ef36a3b9e5b5a9a972a03513a8d49b7054b5c6
git push --tags
```

**Comment by kalekundert** [comment]
No, the 0.19.0 tag is already pushed to master.  The issue is how to create a GitHub release.  Normally I have a GitHub action that takes care of this, but it didn't work for the 0.19.0 release.  Now it basically refuses to run because it thinks the release already happened.  I'll probably have to make the GitHub release manually, but I don't really know how it's created in the first place, so I'm not really sure how to do that.

**Comment by adhawkins** [comment]
Ah, apologies. I missed the fact that the tag was there. That should be enough for me to package it up for Alpine. Working on that now.

I think you can manually create a release just by going into the 'releases' section and creating it, giving it the existing tag. Dunno about adding any assets that you would normally do though.

**Comment by adhawkins** [comment]
Ok, I've done the Alpine release, so the immediate concern for me is over. Feel free to close this if the fact that there's no 'release' isn't an issue for you.
